src/test/java/org/springframework/social/bitbucket/api/impl/GroupPrivilegesTemplateTest.java

```diff
@@ -87,7 +87,7 @@ public void testGetPrivilegedGroupsForRepository() throws Exception {
         assertNotNull(firstPrivilegeGroup.getGroup().getMembers());
         assertTrue(firstPrivilegeGroup.getGroup().getMembers().isEmpty());
         assertEquals("Contractors", firstPrivilegeGroup.getGroup().getName());
-        assertTrue(firstPrivilegeGroup.getRepository().getOwner().isTeam());
+        assertTrue(firstPrivilegeGroup.getRepository().getOwner().getIsTeam());
```

Comment:
Yep, but there is no "get" in front of "is_team" in the JSON format. I would much prefer a "isTeam", or if you really want to follow the Java Bean spec to the letter "getTeam".